cord/client.py

```diff
@@ -384,6 +386,31 @@ def get_project(self):
         """
         return self._querier.basic_getter(Project)
 
+    def add_users(self, user_emails: List[str], user_role: ProjectUserRole):
+        """
+        Add users to project
+
+        Args:
+            user_emails: list of user emails to be added
+            user_role: the user role to assign to all users
+
+        Returns:
+            dictionary list of the added users and project details
+            {project_hash, user_hash, user_email, user_role, user_id}
```

Comment:
Is dictionary list == list of dictionaries? or dictionary of lists? Why not return a 'ProjectUser' dataclass instead?

Reply:
![image](https://user-images.githubusercontent.com/90329688/142875637-7e13895e-898c-4e73-9808-afbe2638d41f.png)